how-to/privileged.md

```diff
@@ -0,0 +1,46 @@
+# Privileged Kata Containers
+
+Kata Containers supports creation of containers that are "privileged" - i.e. have additional capabilities and access
+that isn't normally granted.
```

Comment:
Lines 3 - 4 suggested rewrite: 
Kata Containers supports creation of containers that are "privileged" (i.e. have additional capabilities and access that is not normally granted).